src/control/xml/XmlNode.h

```diff
@@ -20,11 +24,9 @@ class ProgressListener;
 class XmlNode {
 public:
     XmlNode(const char* tag);
-    virtual ~XmlNode();
 
-private:
-    XmlNode(const XmlNode& node);
-    void operator=(const XmlNode& node);
+    XmlNode(const XmlNode& node) = delete;
+    void operator=(const XmlNode& node) = delete;
```

Comment:
Since you are using `std::unique_ptr` those are deleted anyways. But defining them as deleted requires to fulfill the rule of 5. So best would be to remove them. This will implicitly allow to move the object.